crates/scmi/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "vhost-user-scmi"
```

Comment:
In the readme I see examples using `vhost-device-scmi` but the package is `vhost-user-scmi`, maybe it is a leftover, but I agree on renaming all to `vhost-device-scmi` :

```bash
$ git grep vhost-user-scmi
Cargo.lock:name = "vhost-user-scmi"
crates/scmi/Cargo.toml:name = "vhost-user-scmi"
crates/scmi/README.md:# vhost-user-scmi
crates/scmi/README.md:It is tested with QEMU's `-device vhost-user-scmi-pci` but should work
crates/scmi/README.md:      -device vhost-user-scmi-pci,chardev=vscmi,id=scmi \
crates/scmi/kernel/iio-dummy/README.md:industrial I/O devices for testing vhost-user-scmi.
crates/scmi/kernel/iio-dummy/iio_modified_dummy.c: * with vhost-user-scmi:
crates/scmi/kernel/iio-dummy/iio_modified_dummy.h: * with vhost-user-scmi:
crates/scmi/src/devices/common.rs:/// Enumeration of vhost-user-scmi exit codes.
crates/scmi/src/main.rs://! RUST_LOG=debug vhost-user-scmi \
crates/scmi/src/main.rs:            String::from("vhost-user-scmi"),
```

Some of them are not related (e.g. QEMU devices), but just to have an idea ;-)

Reply:
Oh, yes, it should be vhost-device-scmi here and vhost-user-scmi in QEMU. The misnaming should be fixed now.